src/sentry/conf/server.py

```diff
@@ -136,6 +136,7 @@ def env(
 SENTRY_ESCALATION_THRESHOLDS_REDIS_CLUSTER = "default"
 SENTRY_SPAN_BUFFER_CLUSTER = "default"
 SENTRY_ASSEMBLE_CLUSTER = "default"
+SENTRY_UPTIME_DETECTOR_CLUSTER = "default"
```

Comment:
This will need to be adjusted in getsentry before we turn this on yeah? Unless we want to bring sentry down

Reply:
Yep!